src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -131,16 +133,16 @@ if [[ -n "$CROSSCOMPILE" ]]; then
         echo "ROOTFS_DIR not set for crosscompile"
         exit 1
     fi
-    cmake_extra_defines="$cmake_extra_defines -C $1/cross/$build_arch/tryrun.cmake"
-    cmake_extra_defines="$cmake_extra_defines -DCMAKE_TOOLCHAIN_FILE=$1/cross/$build_arch/toolchain.cmake"
+    cmake_extra_defines="$cmake_extra_defines -C $basePath/../../../cross/$build_arch/tryrun.cmake"
```

Comment:
I don't like the script depending on its location. What was wrong with the path being passed in as a parameter?